README.md

```diff
@@ -113,7 +113,11 @@ Left would be created with `nnoremap <silent> <c-h> :TmuxNavigateLeft<cr>`.
 
     let g:tmux_navigator_save_on_switch = 1
 
-This will execute the update command on leaving vim to a tmux pane. Default is Zero
+This will execute the `update` command on leaving vim to a tmux pane. Default is Zero
+
+    let g:tmux_navigator_save_all_buffers_on_switch = 1
```

Comment:
What about using `2` with the existing `g:tmux_navigator_save_on_switch` setting to indicate this behavior?